addons/website/static/src/scss/website.scss

```
@@ -1291,6 +1291,10 @@ header {
         @include o-apply-colors('header-sales_three-custom');
         @include o-add-gradient('menu-secondary-gradient');
     }
+    #wrapwrap header .o_header_sales_three_small_links .btn {
+        // Force small links to be proportional to header font size
+        font-size: .75em !important;
```

You get `12px` with the default font-size, and that's already very small... but if you set the header below `16px` you can easily fall into "really unreadable territory" 😅

I briefly tested `14px` (it was the standard just a few years ago) and you get ~`10px` that's not good for a menu entry 🥲

I have 2 proposals 

1. Use Boostrap default `$small-font-size`(=>  `.875em`)
```suggestion
        font-size: $small-font-size !important;
```

 2. Provide a failsafe value
```suggestion
        font-size: MAX(12px, .75em) !important;
```
I'll let you choose ;)

---

I would choose the second solution, thanks !